spec/ParseGraphQLServer.spec.js

```diff
@@ -9515,6 +9515,118 @@ describe('ParseGraphQLServer', () => {
           }
         });
 
+        it('should support where argument on object field that contains boolean value', async () => {
+          try {
+            const someObjectFieldValue = {
+              foo: { bar: 'baz', qux: true },
+              number: 10,
+            };
+
+            await apolloClient.mutate({
+              mutation: gql`
+                mutation CreateClass($schemaFields: SchemaFieldsInput) {
+                  createClass(input: { name: "SomeClass", schemaFields: $schemaFields }) {
+                    clientMutationId
+                  }
+                }
+              `,
+              variables: {
+                schemaFields: {
+                  addObjects: [{ name: 'someObjectField' }],
+                },
+              },
+              context: {
+                headers: {
+                  'X-Parse-Master-Key': 'test',
+                },
+              },
+            });
+            await parseGraphQLServer.parseGraphQLSchema.schemaCache.clear();
+
+            const schema = await new Parse.Schema('SomeClass').get();
+            expect(schema.fields.someObjectField.type).toEqual('Object');
+
+            const createResult = await apolloClient.mutate({
+              mutation: gql`
+                mutation CreateSomeObject($fields: CreateSomeClassFieldsInput) {
+                  createSomeClass(input: { fields: $fields }) {
+                    someClass {
+                      id
+                    }
+                  }
+                }
+              `,
+              variables: {
+                fields: {
+                  someObjectField: someObjectFieldValue,
+                },
+              },
+            });
```

Comment:
Please follow the test style used in this file.

I can suggest

```suggestion
            const someObjectFieldValue = {
              foo: { bar: 'baz', qux: true },
              number: 10,
            };
            
            const object = new Parse.Object('SomeClass')
            await object.save(someObjectFieldValue)
```

it's shorter 🙂